core/commands/bitswap.go

```diff
@@ -6,17 +6,18 @@ import (
 	"io"
 
 	cmds "github.com/ipfs/go-ipfs/commands"
+	e "github.com/ipfs/go-ipfs/core/commands/e"
```

Comment:
`err` is used all the time and `error` is an important type. `errors` would be confusing because it's an stdlib package. Hence, `e`.